app/controllers/steps_controller.rb

```diff
@@ -514,7 +514,7 @@ def extract_destroy_params(params, update_params)
 
   def load_vars
     @step = Step.find_by_id(params[:id])
-    @protocol = @step.protocol
+    @protocol = @step&.protocol
```

Comment:
what will happen in case when @protocol is nil?

Reply:
There is check later:
````
    unless @protocol
      render_404
    end
````